apis/v1alpha1/defaulting.go

```diff
@@ -130,11 +163,43 @@ func (in *GreptimeDBCluster) defaultFlownodeSpec() *FlownodeSpec {
 		ComponentSpec: ComponentSpec{
 			Template: &PodTemplateSpec{},
 			Replicas: pointer.Int32(DefaultReplicas),
+			Logging:  &LoggingSpec{},
 		},
 		RPCPort: DefaultRPCPort,
 	}
 }
 
+func (in *GreptimeDBCluster) defaultMonitoringStandaloneSpec() *GreptimeDBStandaloneSpec {
+	standalone := new(GreptimeDBStandalone)
+	standalone.Spec = *standalone.defaultSpec()
+
+	if image := in.GetBaseMainContainer().GetImage(); image != "" {
+		standalone.Spec.Base.MainContainer.Image = image
+	} else {
+		standalone.Spec.Base.MainContainer.Image = DefaultGreptimeDBImage
+	}
+
+	standalone.Spec.Version = getVersionFromImage(standalone.Spec.Base.MainContainer.Image)
+
+	if osp := in.GetObjectStorageProvider(); osp != nil {
+		standalone.Spec.ObjectStorageProvider = osp.DeepCopy()
+
+		if root := osp.GetS3Storage().GetRoot(); root != "" {
+			standalone.Spec.ObjectStorageProvider.S3.Root = fmt.Sprintf("%s/monitoring", root)
```

Comment:
There should be no need to add the `monitoring` directory ？

Reply:
The monitoring data can be in the same directory as the cluster.